pkg/scheduler/api/unschedule_info.go

```diff
@@ -56,13 +58,24 @@ func (f *FitErrors) SetNodeError(nodeName string, err error) {
 	default:
 		fe = &FitError{
 			NodeName: nodeName,
-			Reasons:  []string{obj.Error()},
+			Status:   []*Status{{Code: Error, Reason: obj.Error()}},
 		}
 	}
 
 	f.nodes[nodeName] = fe
 }
 
+// NotHelpfulForPreemptNodes returns the set of nodes that has no help from preempting pods from it
```

Comment:
suggest change function name to `getUnschedulableAndUnresolvableNodes`. This function might be used for more places besides preemption.

Reply:
Done